adding github actions to create the dtbs and adding the x1e78100 to the dts

pixel844 opened the pull request:
added support to the x1e78 for acer swift 14 ai while adding some github actions to spare

pixel844 committed:
Create mainu.yml

pixel844 committed:
Create main.yml

pixel844 committed:
Delete .github/workflows/main.yml

pixel844 committed:
Rename mainu.yml to main.yml

pixel844 committed:
Update main.yml

pixel844 committed:
Update x1p64100-acer-swift-sf14-11.dts

pixel844 committed:
Merge branch 'jglathe:jg/ubuntu-qcom-x1e-6.17' into jg/ubuntu-qcom-x1e-6.17

jglathe commented:
Hi @pixel844, thank you for the PR. The main change is changing the compatible string... this one is debatable. The one I show/ship has "lenovo,thinkpad-t14s" in the string, not "qcom,x1e78100". Which also is basically only t14s. But, the reason for this is, should you pair the dt with an upstream kernel, qcom_scm would still find a compatible it knows and has in its list of compatible strings, and thats the "lenovo,thinkpad-t14s" but not "qcom,x1e78100" nor any other string that is in the string of the acer dts. Leading to "efivars not supported", and install fails. Not clean-ish, but works well.
As for the github actions: thank you, will take a look.

pixel844 committed:
Update x1p64100-acer-swift-sf14-11.dts

pixel844 committed:
Update x1p64100-acer-swift-sf14-11.dts

pixel844 commented:
hey thanks for the info, Im quite suprised that "qcom,x1e78100" is not recognized by the upstream kernel, maybe someday.

tuxedo-ggo committed:
dt-bindings: vendor-prefixes: Add prefix for TUXEDO Computers GmbH

TUXEDO Computers GmbH is a German supplier for computers.

Signed-off-by: Georg Gottleuber <ggo@tuxedocomputers.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>

tuxedo-ggo committed:
dt-bindings: arm: qcom: Add TUXEDO Computers device

Introduce new binding for the TUXEDO Elite 14 Gen1 laptop with X1E78100
SoC.

Signed-off-by: Georg Gottleuber <ggo@tuxedocomputers.com>
Reviewed-by: Krzysztof Kozlowski <krzk@kernel.org>

tuxedo-ggo committed:
dt-bindings: vendor-prefixes: Add Medion AG

Add Medion AG, a German electronics company, to the list of vendor
prefixes.

Link: https://www.medion.com/

Signed-off-by: Georg Gottleuber <ggo@tuxedocomputers.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@oss.qualcomm.com>

tuxedo-ggo committed:
dt-bindings: arm: qcom: Add Medion SPRCHRGD device

Introduce new binding for the Medion SPRCHRGD 14 S1 notebook with
X1E78100 SoC.

Signed-off-by: Georg Gottleuber <ggo@tuxedocomputers.com>

tuxedo-ggo committed:
arm64: dts: qcom: Add dts for Medion SPRCHRGD 14 S1

Initial support for the Medion SPRCHRGD 14 S1, which is based on the
Qualcomm Snapdragon X Elite SoC (X1E78100).

Working:
* Touchpad
* Keyboard
* eDP
* NVMe
* USB Type-C port
* USB-C DP altmode
* HDMI-A port
* WiFi
* Bluetooth
* GPU
* Video decoding
* USB Type-A
* Audio, speakers, microphones
        - 4x speakers.
        - 2x dmic
        - headset
* Camera
* Fingerprint reader

Co-developed-by: Srinivas Kandagatla <srini@kernel.org>
Signed-off-by: Srinivas Kandagatla <srini@kernel.org>
Co-developed-by: Ettore Chimenti <ettore.chimenti@linaro.org>
Signed-off-by: Ettore Chimenti <ettore.chimenti@linaro.org>
Signed-off-by: Georg Gottleuber <ggo@tuxedocomputers.com>

tuxedo-ggo committed:
firmware: qcom: scm: QSEECOM on Medion SPRCHRGD 14 S1

This is needed to access EFI variables from Linux.

Signed-off-by: Ettore Chimenti <ettore.chimenti@linaro.org>
Signed-off-by: Georg Gottleuber <ggo@tuxedocomputers.com>

jayaddison committed:
arm64: dts: qcom: follow `hamoa` rename

Relates-to commit 8c0b058ab5983a4be6690a76be9b0294853e8e55.

Signed-off-by: James Addison <jay@jp-hosting.net>

jglathe committed:
Merge pull request #55 from jayaddison/jka/ubuntu-qcom-x1e-6.17-medion-s1

dts: add Medion SPRCHRGD 14 S1

pixel844 committed:
Merge branch 'jglathe:jg/ubuntu-qcom-x1e-6.17' into jg/ubuntu-qcom-x1e-6.17
